modules/govuk_crawler/manifests/config.pp

```diff
@@ -0,0 +1,36 @@
+#
+# [*aws_access_key*] access key for AWS
+# [*aws_secret_key*] secret for chosen AWS key
+#
+# Will be set using environment variables, see:
+# http://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html#cli-environment
+#
+
+class govuk_crawler::config (
+  $aws_access_key = undef,
+  $aws_secret_key = undef,
+) {
+  $env_name = 's3_sync_mirror'
+
+  file { ["/etc/govuk/${env_name}", "/etc/govuk/${env_name}/env.d"]:
+    ensure  => present,
+    purge   => true,
+    recurse => true,
+    force   => true,
```

Comment:
You'll need a `mode` here as the `File` underneath will set a default that might get picked up.